js/controller.js

```diff
@@ -823,6 +844,13 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.DISCOVERY_API.SEND_DISPLAY_EVENT, eventData);
     },
 
+    sendVideoQualityChangeEvent: function(selectedContentData) {
+      this.state.videoQualityOptions.selectedBitrate = {
+        "id": selectedContentData.id
+      };
+      this.mb.publish(OO.EVENTS.SET_TARGET_BITRATE, selectedContentData.selectedBitrate);
```

Comment:
I know it's only a mock, but selectedContentData only has 'id' property here, maybe we should publish selectedContentData.id then.